pkgs/games/multimc/default.nix

```diff
@@ -18,7 +18,16 @@ in stdenv.mkDerivation rec {
   enableParallelBuilding = true;
 
   postInstall = ''
+    mkdir -p $out/share/{applications,pixmaps}
+    cp ../application/resources/multimc/scalable/multimc.svg $out/share/pixmaps
```

Comment:
What is $out/share/pixmaps for? Am curious

Reply:
Stuff for like apps like this with pixmaps... with java? Idk :joy: